.github/workflows/go-it-os.yaml

```diff
@@ -21,11 +18,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Get Helm version
```

Comment:
:wrench: You can install Helm via asdf, the RedHat installer doesn't install a special version of helm but was just a convenience.

Reply:
That's much better :grin: 
I will change it then.

Thanks for the feedback :ok_hand: